packages/common/client-utils/src/bufferNode.ts

```diff
@@ -8,7 +8,7 @@
  * exposing the entirely of Node's typings.  This should match the public interface
  * of the browser implementation, so any changes made in one should be made in both.
  *
- * @internal
```

Comment:
@Josmithr What does "internal" mean here? Does it mean there is no semver stability guarantee? if so, tree depends on this package using a semver (^) constraint, so it shouldn't use these types.

My understanding is that this entire package is "fluid-internal" (see the package's scope), which means it only gets updates/fixes to support other fluid packages. Given that, I think all these APIs should be marked as @public, since I think we need (and want) standard semver guarantees, and they should be available to all users of this package.

I think @alpha would be bad here, since that implies they are not stable for semver, and since these APIs show up in use of fluid handles in stable code, I think we actually do need to ensure they are stable.

Reply:
Yeah, you're right. Sorry, I misunderstood which package we were modifying here. Yeah, public is probably the better call for these.

Reply:
Actually, I don't see APIs other than SharedTree2 leaking these into their package public APIs. Maybe leaving these as internal makes sense and we should adjust those APIs?

@Abe27342 Do you have any insight here? Would it be easy to tweak IBinaryCodec to not leak these types?

Reply:
ehh--I don't think it's particularly easy to change at that layer. Ideally tree2 wouldn't leak the entire codec abstraction, which would handle it. The reason it's exposed right now is due to `FieldChangeHandler`.

Reply:
@CraigMacomber @taylorsw04 should we file work to fix this on the tree side then?

Reply:
This is rather difficult, but might be worth it (also helps hide a lot of other things from the APi surface). I'm going to try an approach and see if it works

Reply:
Done.